cypress/e2e/opting-out.cy.ts

```diff
@@ -112,6 +112,40 @@ describe('opting out', () => {
             pollPhCaptures('$snapshot').then(assertThatRecordingStarted)
         })
 
+        it('can start recording when starting disabled', () => {
+            cy.posthogInit({ disable_session_recording: true })
+
+            assertWhetherPostHogRequestsWereCalled({
+                '@recorder': false,
+                '@decide': true,
+                '@session-recording': false,
+            })
+
+            cy.get('[data-cy-input]')
+                .type('hello posthog!')
+                .then(() => {
+                    cy.phCaptures().then((captures) => {
+                        expect(captures || []).to.deep.equal(['$pageview'])
+                    })
+                })
+
+            cy.posthog().invoke('startSessionRecording')
+
+            assertWhetherPostHogRequestsWereCalled({
+                '@recorder': true,
+                '@decide': true,
+                // no call to session-recording yet
```

Comment:
Should we assert that it does eventually get called?

Reply:
oh, wait... i didn't finish this yet 🙈 

was distracted by my sandwich arriving 🤣 🍽️